### DISCUSSION TITLE
aspect ratio improvements (#2171) error

### DISCUSSION THREAD

**Discussion by VictorBDias** [sampotts/plyr#2177] [Q&A]
![Screenshot from 2021-04-19 10-38-26](https://user-images.githubusercontent.com/30843291/115245970-d96d8180-a0fb-11eb-8da0-bc6b01ff6b97.png)

any ideas?

**Comment by sampotts** [comment]
See https://github.com/sampotts/plyr/issues/2174. This should really be an issue. 👍🏼
